Jenkinsfile

```diff
@@ -350,6 +350,18 @@ tar -xzf packaging/target/apache-hive-*-nightly-*-src.tar.gz
       }
     }
   }
+  branches['javadoc-check'] = {
+    executorNode {
+      stage('Prepare') {
+          loadWS();
+      }
+      stage('Generate javadoc') {
+          sh """#!/bin/bash -e
+mvn clean install javadoc:javadoc javadoc:aggregate -DskipTests
```

Comment:
this is not good; please look how other parts of this file are using maven.

Reply:
I had to exclude some of the itest modules, because those have some weird compilation issues when generating javadoc. Since we are not really intereseted in the itest javadoc, I think it is good enough for now.

What do you think?